### PR TITLE
Fc 282 mvs assets (images, case studies, video and logos) are on pre prod site

### PR DESCRIPTION
**Pull request by desderry**

### CODE REVIEW

File: src/SFA.DAS.Campaign.Web/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -50,65 +50,54 @@
 
     <div class="grid-row hero-banner hero-banner--apprentice hero-banner--right-aligned">
         <div class="grid-column-full">
-            <h2 class="hero-banner__caption">
+            <div class="hero-banner__caption">
```
Comment (jamesmking) [inline src/SFA.DAS.Campaign.Web/Views/Home/Index.cshtml]: This should be a H2 now

File: src/SFA.DAS.Campaign.Web/Views/Home/Index.cshtml [hunk 2]
```diff
@@ -50,65 +50,54 @@
...
-                    <div class="hero-banner__cite">Sarah Bloggs <span class="hero-banner__cite-detail">33, Designer, Manchester</span></div>
-           
-                    <a asp-controller="RealStories" asp-action="Apprentice" class="hero-banner__link">
-                        View more apprentice stories
-                    </a>
-                </div>
             </div>
-            <span class="hero-banner__background"></span>
+            <div class="grid-column-one-half hero-banner__content-main">
+                <h3 class="hero-banner__header">Get the confidence, grab the career.</h3>
```
Comment (jamesmking) [inline src/SFA.DAS.Campaign.Web/Views/Home/Index.cshtml]: This should be a P tag

File: src/SFA.DAS.Campaign.Web/Views/Home/Index.cshtml [hunk 3]
```diff
@@ -50,65 +50,54 @@
...
-                    <p class="hero-banner__header">Work with training providers to create qualified, highly employable staff.</p>
-                    <p class="hero-banner__copy">
-                        "If you're starting to offer apprenticeships for the first time, 
-                    talking to other people in your industry about training providers is an invaluable resource."
-                    </p>
-                    <p class="hero-banner__cite">
-                    Bindhi Ashfield
+<div class="grid-row hero-banner hero-banner--employer">
+    <div class="grid-column-full">
+        <div class="hero-banner__caption">
```
Comment (jamesmking) [inline src/SFA.DAS.Campaign.Web/Views/Home/Index.cshtml]: This should be a H2

File: src/SFA.DAS.Campaign.Web/Views/Home/Index.cshtml [hunk 4]
```diff
@@ -50,65 +50,54 @@
...
+            <span class="hero-banner__caption-icon"></span>
+            <span class="hero-banner__caption-text">Real employer stories</span>
+        </div>
+        <div class="grid-row hero-banner__content">
+            <div class="grid-column-one-half  hero-banner__icon-play-container">
+                <a class="hero-banner__icon-play-mobile" href="#player-2" id="play-mobile-2"></a>
+                <a class="hero-banner__icon-play-desktop" href="#player-2" id="play-desktop-2"></a>
+            </div>
+            <div class="grid-column-one-half hero-banner__content-main">
+                <h3 class="hero-banner__header">A real win-win for businesses, large and small.</h3>
```
Comment (jamesmking) [inline src/SFA.DAS.Campaign.Web/Views/Home/Index.cshtml]: Needs to be a P tag

**Comment by jamesmking** [comment]
The markup on the homepage was changed as a result of the accessibility report, I think your changes haven’t merged well with the changes I made

File: src/SFA.DAS.Campaign.Web/Views/Home/Index.cshtml (hunk 1 above)
Comment (desderry) [reply src/SFA.DAS.Campaign.Web/Views/Home/Index.cshtml]: done

**Comment by desderry** [comment]
i've now updated tags as per comments, should be ok now

**Pull request merged**